x-pack/plugins/code/public/components/admin_page/project_tab.tsx

```diff
@@ -27,21 +28,39 @@ const NewProjectButton = styled(EuiButton)`
   margin-top: 1.5rem;
 `;
 
-// TODO(qianliang)
+enum SortOptionsValue {
+  alphabetical_asc = 'alphabetical_asc',
+  alphabetical_desc = 'alphabetical_desc',
+  updated_asc = 'updated_asc',
+  updated_desc = 'updated_desc',
+  recently_added = 'recently_added',
+}
+
+const sortFunctions: { [k: string]: (a: Repository, b: Repository) => number } = {
+  [SortOptionsValue.alphabetical_asc]: (a: Repository, b: Repository) =>
+    a.name! > b.name! ? 1 : -1,
```

Comment:
nit: I think `a.name! > b.name!` should work?

Reply:
type would be wrong